scss/tabstrip/_theme.scss

```diff
@@ -62,30 +62,30 @@ $tabstrip-content-border: $panel-bg !default;
     .k-tabstrip-top {
         > .k-tabstrip-items {
             .k-item.k-state-active {
-                border-bottom-color: $tabstrip-item-selected-bg;
+                border-bottom-color: $tabstrip-item-selected-border;
```

Comment:
@joneff Totally missed this response - if it is required to be the 'bg' value here what's the point of having the selected border color?